packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.scss

```diff
@@ -17,6 +17,7 @@
   position: relative;
   ::slotted(fluent-tree-item) {
     --tree-item-nested-width: 2em;
+    --card-height: auto;
```

Comment:
this needs to be set to auto higher up in the DOM tree, as it needs to be applied to the `fluent-card` and not the tree-item

Reply:
@gavinbarron fixed it up :)